test/test_transforms_v2.py

```diff
@@ -5783,3 +5823,18 @@ def test_errors(self):
         with pytest.raises(ValueError, match="Number of boxes"):
             different_sizes = {"bbox": good_bbox, "labels": torch.arange(good_bbox.shape[0] + 3)}
             transforms.SanitizeBoundingBoxes()(different_sizes)
+
+        with pytest.raises(ValueError, match="canvas_size cannot be None if bounding_boxes is a pure tensor"):
```

Comment:
Nit: I think for the other functional / transform combinations we don't mix the error tests, i.e. one test for the transform and one for the functional / kernel.